torch/tensor.py

```diff
@@ -424,7 +430,7 @@ def istft(self, n_fft, hop_length=None, win_length=None, window=None,
                 window=window, center=center, normalized=normalized, onesided=onesided, length=None
             )
         return torch.istft(self, n_fft, hop_length, win_length, window, center,
-                           normalized, onesided, length)
+                           normalized, onesided, return_complex=return_complex)
```

Comment:
What happened to passing length here?

Reply:
Good catch. It seems the method variant isn't tested very well. I'll add it to one of the tests in this PR.

Reply:
Just ping me when it's ready.

Reply:
I also notice the torch_function calls were missing arguments as well so I've updated those.

Reply:
@mruberry this should be ready now, assuming CI passes.